### ISSUE TITLE
Discovery: Where are Full Rich text editors used in the CMS

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#12811]
## Description
Looking at how we might limit editor created heading violations coming from the CMS, we want to see where all the full rich text editors are being used. On a by product basis, if we can see where this particular WYSIWYG is available (and therefore where editors are able to add headings), and what the purpose of each of these WYSIWYGs is, we can then start to think about which of these we might be able to remove or update and use a different field type instead.

## Acceptance Criteria
- [x] Determine, by product, where full rich text editors are used in the CMS
- [x] Determine the purpose of each rich text editor above
- [x] Provide validations of use/recommendations for alternatives

### Team
Please check the team(s) that will do this work.

- [x] `CMS Team`
- [ ] `Public Websites`
- [ ] `Facilities`
- [ ] `User support`

**Comment by swirtSJW** [comment]
This is not surfaced yet by the drupal content model View.  The data is in there, we just are not surfacing it yet.

It is in the spectool here https://airtable.com/appSg80z9nVWoesxn/tblPdSdScGDeV1Y6f/viwMsHjkkJQdlbDSP?blocks=bipz2F2iCJ6xlOlNM  Scroll down to "text formatted, long "

**Comment by swirtSJW** [comment]
Actually I take that back, we can surface it in the CMS  https://prod.cms.va.gov/admin/reports/content-model/fields?field_name=&field_type=text_long&entity_type=&bundle=&label=

and 

https://prod.cms.va.gov/admin/reports/content-model/fields?field_name=&field_type=text_with_summary&entity_type=&bundle=&label=

**Comment by laflannery** [comment]
Note from when I was working - there is no filter for which type for rich text editor. Do we think this  would be useful for any case other than the one I worked on in this particular ticket (which was no really an issue not having if because there were not hundreds of instances)?
- I feel like no, but I still wanted to bring it up just in case
- Also this filter would only apply to one field type, also a solid reason not to add it

**Comment by laflannery** [comment]
I don't think this is surfacing all fields, for example on [this page](https://staging.cms.va.gov/admin/structure/taxonomy/manage/facility_supplemental_status/add) there are 2 rich text editors, Description and Guidance. The Guidance field is in the link you gave me above, the Description field is not. Why wouldn't that be here?
![image](https://user-images.githubusercontent.com/106678594/225429304-bd36dedf-d87f-4502-aae6-a2742b9a0b15.png)

**Comment by laflannery** [comment]
Something I have noticed that is out of scope for this but I think its worth noting is the various paragraph types we have, and are they really all necessary? For example:
- Q&A. We have 2 paragraph types:
  - Q&A:
     - Question field (text) and answer field (Full rich text)
     - Ability to add as part of the answer: Rich text, Accordion group, Process list, Number callout, Alert, React widget, and/or Table
  - Q&A Section:
    - Section header field (text) and Section intro (plain text)
    - Check box to display as accordion group or not
    - Question field (text) and answer field (Full rich text)
    - Ability to add as part of the answer: Rich text, Accordion group, Process list, Number callout, Alert, React widget, and/or Table
  - Q&A group:
    - Header field (text) and Introduction (Limited rich text)
    - Check box to display as accordion group or not
    - Select to pull already existing Q&As

It just seems to me that for consistency, some of this could be streamlined
- Section header vs Header
- Section intro vs introduction (and which type of field should we have here?)

This isn't the only instance but it's just something I was noticing and wanted to note
